common/spec/importers/firefoxCsvImporter.spec.ts

```diff
@@ -1,8 +1,8 @@
-import { FirefoxCsvImporter as Importer } from 'jslib-common/importers/firefoxCsvImporter';
+import { FirefoxCsvImporter as Importer } from '../../src/importers/firefoxCsvImporter';
```

Comment:
Do we have to do this? I'd prefer it staying the way it is. The clients using jslib also work with referrering to jslib-common package

Reply:
Added `src` as an alias.